app/Services/SprintChartService.php

```diff
@@ -121,6 +127,8 @@ private function buildNewSprintFromJiraResponse(array $sprintResponse, array $is
                 'id' => $sprintResponse['id'],
                 'name' => $sprintResponse['name'],
                 'board_id' => $sprintResponse['originBoardId'],
+                'start_date' => $sprintStartDate,
+                'end_date' => $sprintEndDate,
                 'length' => $sprintLength,
```

Comment:
To make these match the others
```suggestion
                'length' => $sprintLength,
                'start_date' => $sprintStartDate,
                'end_date' => $sprintEndDate,
```